drivers/src/nrfx_saadc.c

```diff
@@ -626,4 +626,21 @@ void nrfx_saadc_limits_set(uint8_t channel, int16_t limit_low, int16_t limit_hig
         nrf_saadc_int_enable(int_mask);
     }
 }
+
+void nrfx_enable_adc_chan(int chan, nrf_saadc_input_t   pselp,
```

Comment:
Please change namespace to nrfx_saadc by changing name to:
`nrfx_saadc_channel_enable`